pages/platform/ai/ecosystem_02_lettria_models/guide.en-gb.md

```diff
@@ -0,0 +1,346 @@
+---
+title: AI Partners Ecosystem - Lettria - Models concept
+excerpt: Learn how to use Lettria models
+updated: 2023-07-10
+---
+
+**Last updated 10th July, 2023.**
+
+## Objective
+
+OVHcloud offers different Artificial Intelligence services through its AI Partners Ecosystem. You will benefit from a catalogue of **ready-to-use applications** provided by our partners which you will easily be able to deploy according to your needs through **AI Deploy**.
+
+**Lettria is an OVHcloud partner that offers AI services dedicated to text. This guide will provide a detailed understanding of how Lettria services work.**
+
+> [!primary]
+>
+> To find out more about Lettria billing, launching and capabilities, please refer to this [guide](/pages/platform/ai/ecosystem_01_lettria_billing_features_capabilities).
+>
+
+## Introduction
+
+**Lettria** is a start-up specialized in **NLP** (Natural Language Processing). The platform enables all organizations, from start-ups to large corporations, to perform textual analysis on their data to take the best strategic decisions.
+
+Lettria provides text understanding models that allow users to easily identify and extract key information from their text. This method relies on artificial intelligence and NLP techniques to understand the sentiment or emotions of a text.
+
+## Lettria’s Sentiment Analysis API
+
+Our Sentiment Analysis API, powered by Deep Learning, applies advanced algorithms to examine and predict sentiment values from -1 to 1, offering key insights into the emotional tone of text.
+
+The Sentiment Analysis API facilitates sentiment assessment in diverse content, including:
+
+- customer feedback
+- social media posts
+- product reviews
+
+By attributing a numerical value to the sentiment of text, it provides an understanding of the overall positivity or negativity of the content.
+
+The API enables objective measurement of sentiment in large data sets, aiding informed decision-making and timely actions. It caters to needs such as brand reputation monitoring, market research and customer experience optimization, empowering you with actionable insights.
+
+Our API delivers sentiment values from -1 (signifying strong negative sentiment) to 1 (indicating strong positive sentiment), facilitating comprehensive understanding of sentiment intensity. This allows you to identify sentiment polarity and measure its degree in the text.
+
+The API's user-friendly design allows effortless integration into your applications, unlocking the capabilities of sentiment analysis. Enhance customer satisfaction, take data-driven decisions and optimize business strategies with sentiment analysis. Discover and understand the true sentiment behind text, revolutionizing your audience engagement.
```

Comment:
The 2nd and 3rd sentences "sound" very marketing-oriented, maybe too much for documentation purposes.

Reply:
Indeed, I agree with you... But it's the partner's doc and we didn't write that part of the guide